### PR TITLE
Peer Feedback Review: To Do List

### PR DESCRIPTION
**Pull request by AmanWorku**
- Add previous project files.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  font-size: 1.2em;
+  font-family: "Quicksand", sans-serif;
+  font-style: italic;
+  opacity: 0.7;
+}
+
+form .task-input:focus {
+  outline: none;
+  font-size: 1em;
+  font-family: "Quicksand", sans-serif;
```
Comment (felixDev22) [inline src/style.css:93]: Great job so far. 👍  Just a slight change -  My suggestion is you may avoid font-family repeats - for better CSS best practices. You could create a root with the property font family.
and make your code look good. 💯 😄

File: src/modules/crud.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+const clearCompleted = document.querySelector('.clear');
+
+clearCompleted.addEventListener('click', () => {
+  const storedData = localStorage.getItem('data');
+  let listArr = JSON.parse(storedData);
+  listArr = listArr.filter((element) => element.completed === false);
+  for (let i = 0; i < listArr.length; i += 1) {
+    listArr[i].index = i + 1;
+  }
```
Comment (felixDev22) [inline src/modules/crud.js:144]: Great job, clean design and interactive - please check the clear button is not working - otherwise all looks good. 👍 💯

**Review by felixDev22 — COMMENTED** [review]
You have done a great job, your code is clean and precise. Keep it up. 😄 😄 👍 

- [ ] Just noted 2 slight issues...overall your work is fantastic 👍 🎉 💯

File: src/style.css (hunk 1 above)
Comment (AmanWorku) [reply src/style.css:93]: Thank you so much, I'll do it accordingly.

File: src/modules/crud.js (hunk 2 above)
Comment (AmanWorku) [reply src/modules/crud.js:144]: Thank you so much.

Comment (AmanWorku) [reply src/modules/crud.js:144]: - The clear completed button works when you check one or more tasks and then when you click they will disappear. Please check that.

**Review by felixDev22 — APPROVED** [review]
STATUS: APPROVED!!! 🎉 🎉 🎉 

Good job !!!
To Highlight! 👏 🟢

✔️ All linters passing
✔️ Professional Readme
✔️ Proper use of JS, HTML & CSS
✔️ Clean and interactive app

Your project is complete! 👍   Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 🎉 🎉

**Pull request merged**